app/src/main/java/com/android/sample/ui/createRecipe/RecipeListInstructionsScreen.kt

```diff
@@ -146,12 +169,22 @@ fun RecipeListInstructionsContent(
                 })
           }
         }
-
+    val context = LocalContext.current
     // Fixed button at the bottom
     PlateSwipeButton(
         text = stringResource(R.string.next_step),
         modifier = Modifier.align(Alignment.CenterHorizontally).testTag(NEXT_STEP_BUTTON),
-        onClick = { navigationActions.navigateTo(Screen.CREATE_RECIPE_ADD_IMAGE) },
+        onClick = {
```

Comment:
Dude, this is awesome! That check for existing instructions is super smart, it makes sure users don’t skip adding one, and the toast message is a nice touch to guide them. Really thoughtful work here!

Reply:
Thanks !